tp_models/users/controller.py

```diff
@@ -37,7 +37,7 @@ def add_user():
         return e.message, e.status
 
     token = generate_confirmation_token(email)
-    send_email(email, f"Confirmation Email for {username}", f"Click here to confirm! -> http://127.0.0.1:5000/confirm/{token}")
+    send_email(email, f"Confirmation Email for {username}", f"Click here to confirm! -> http://127.0.0.1:5000/users/confirm/{token}")
```

Comment:
Is there a reason not to include token generation and sending email confirmation within the `try` block? And handle appropriate exceptions that could occur.

Reply:
done